geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/GeoMesaMetadataStats.scala

```diff
@@ -45,6 +46,16 @@ class GeoMesaMetadataStats(val ds: AccumuloDataStore, statsTable: String)
 
   private val metadata = new AccumuloBackedMetadata(ds.connector, statsTable, new StatsMetadataSerializer(ds))
 
+  private val compactionScheduled = new AtomicBoolean(false)
+  private val lastCompaction = new AtomicLong(0L)
+
+  executor.scheduleWithFixedDelay(new Runnable() {
+    override def run(): Unit = if (compactionScheduled.compareAndSet(true, false) &&
+        lastCompaction.get < DateTime.now(DateTimeZone.UTC).minusHours(1).getMillis) {
+      compact()
+    }
+  }, 1, 1, TimeUnit.HOURS)
```

Comment:
@jahhulbert-ccri @anthonyccri thoughts on this?  

Should this be configurable and/or optional?